src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -114,6 +116,20 @@ winrt::com_ptr<Profile> Profile::CopySettings(winrt::com_ptr<Profile> source)
     profile->_BackgroundImageAlignment = source->_BackgroundImageAlignment;
     profile->_ConnectionType = source->_ConnectionType;
 
+    if (source->_UnfocusedAppearance)
+    {
+        if (source->_UnfocusedAppearance.value() == nullptr)
+        {
+            profile->_UnfocusedAppearance = nullptr;
+        }
+        else
+        {
+            winrt::com_ptr<AppearanceConfig> appearanceImpl;
+            appearanceImpl.copy_from(winrt::get_self<AppearanceConfig>(source->_UnfocusedAppearance.value()));
+            //profile->_UnfocusedAppearance = *AppearanceConfig::CopyAppearance(appearanceImpl);

```

Comment:
This line causes a compiler error:

'initializing': cannot convert from 'winrt::Microsoft::Terminal::Settings::Model::implementation::AppearanceConfig *const ' to 'winrt::impl::abi<winrt::Microsoft::Terminal::Settings::Model::IAppearanceConfig,void>::type *'	Microsoft.Terminal.Settings.Model.Lib

Reply:
```suggestion
            profile->_UnfocusedAppearance = AppearanceConfig::CopyAppearance(appearanceImpl).as<IAppearanceConfig>();
```
Maybe try something like this?

Reply:
Still causes the same error!

Reply:
Okay fixed now!